modules/dxp/apps/commerce/commerce-shop-by-diagram-web/src/main/resources/META-INF/resources/render/list/entry/view.jsp

```diff
@@ -46,6 +46,12 @@ CSDiagramCPTypeDisplayContext csDiagramCPTypeDisplayContext = (CSDiagramCPTypeDi
 						</span>
 					</a>
 				</p>
+
+				<p class="card-text" title="<%= cpCatalogEntry.getShortDescription() %>">
+					<span class="text-truncate-inline">
+						<span class="text-truncate"><%= cpCatalogEntry.getShortDescription() %></span>
+					</span>
+				</p>
 			</div>
```

Comment:
hey @dsitu we don't need this. We will leave customer make customization by implementing their custom renderer.